exampleApp/package.json

```diff
@@ -13,7 +13,7 @@
     "test:migration:run": "ts-node -r dotenv-safe/config node_modules/.bin/typeorm migration:run dotenv_config_path=.env.test",
     "test:schema:drop": "ts-node -r dotenv-safe/config node_modules/.bin/typeorm schema:drop dotenv_config_path=.env.test",
     "start:debug": "nodemon --config nodemon-debug.json",
-    "test:unit": "ts-node -r dotenv-safe/config node_modules/.bin/jest * --maxWorkers=4 dotenv_config_path=.env.test"
+    "test:unit": "source .env.test && ts-node -r dotenv-safe/config node_modules/.bin/jest * --maxWorkers=4 dotenv_config_path=.env.test"
```

Comment:
This is not ran when running `yarn test` in the root folder (there's actually no test in exampleApp). On the other hand, the tests in the library use the `TestTypeOrmModule`, in which the connection options are still hardcoded, we need to use env variables there